licenses.d2iq.yaml

```diff
@@ -224,11 +224,11 @@ resources:
     sources:
       - ref: ${image_tag}
         url: https://github.com/mesosphere/kubetunnel
-  - container_image: docker.io/mesosphere/nkp-pulse-management:v0.1.2
+  - container_image: pullthrough.infra.nkp.sh/mesosphere/nkp-pulse-management:v0.2.6
```

Comment:
See [NCN-106147](https://jira.nutanix.com/browse/NCN-106147) for the reasoning behind this, this will be reverted to `docker.io` before release.